bcos-gateway/bcos-gateway/libp2p/P2PMessage.cpp

```diff
@@ -210,6 +231,25 @@ bool P2PMessage::encode(bytes& _buffer)
     return true;
 }
 
+/// compress the payload data to be sended
+bool P2PMessage::compress(std::shared_ptr<bytes> compressData)
+{
+    if (m_payload->size() <= bcos::gateway::c_compressThreshold)
+    {
+        return false;
+    }
+
+    size_t compressSize =
+        ZstdCompress::compress(ref(*m_payload), *compressData, bcos::gateway::c_zstdCompressLevel);
+    if (compressSize < 1)
```

Comment:
Use Micro `CompressError` to instead of `<1`.
eg.
`if(compressSize == CompressError)`

Reply:
fixed